src/custom_modules/DAO_modules/donations.ts

```diff
@@ -810,7 +810,7 @@ async function add(
   registeredDate = null,
   externalPaymentID = null,
   metaOwnerID = null,
-) {
+): Promise<number> {
```

Comment:
Could be null or undefined? Wha happens if no donor is found?

Reply:
It seems to me it would throw an error if there's no donor found

Reply:
Oh, yes, you are totally right, did not see that 👍